src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/UnixExportProvider.cs

```diff
@@ -151,7 +151,11 @@ private byte[] ExportPfx(SafePasswordHandle password)
             }
             finally
             {
-                password.DangerousRelease();
+                if (gotRef)
```

Comment:
This isn't necessary. You'll never get here if gotRef is false because the DangerousAddRef call is before the try block.

Reply:
I will move the DangerousAddRef to the try block then.